portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,20 @@
 /**
  * Adds a random greeting to the page.
  */
+
 function addRandomGreeting() {
```

Comment:
nit: Since this is no longer adding a random greeting, it might be worth updating the function name and the documentation above.